frontend/passkeys.js

```diff
@@ -20,14 +20,19 @@ export async function registerPasskey(csrfToken) {
             passkeyName: passkeyName
         });
     } catch (err) {
-        console.error('Error during passkey registration:', err);
-        M.toast({ html: 'Failed to register passkey. Please try again.', classes: 'rounded red' });
+        if (err.name === 'InvalidStateError') {
+            console.warn('Passkey already registered:', err);
+            M.toast({html: 'This passkey has already been registered.', classes: 'rounded orange'});
+        } else {
+            console.error('Error during passkey registration:', err);
+            M.toast({html: 'Failed to register passkey. This may be because this passkey has already been registered or it could be a transient issue, in which case please try again.', classes: 'rounded red'});
```

Comment:
The message is very long and it might not stay up long enough to read it all. We could increase the display time and/or shorten the message to something like this?
```suggestion
            M.toast({html: 'Registration failed: this passkey may have already been registered, or it could be a transient issue in which case please try again.', classes: 'rounded red'});
```

Reply:
When we include the possibility that the registration has been cancelled we'll have to start using footnotes!